lib/utils/patterns.ts

```diff
@@ -390,9 +411,9 @@ export async function getOnlineTargetUuid(
 	// otherwise, it may be a device OR an application...
 	try {
 		logger.logDebug(
-			`Fetching application by name ${applicationOrDevice} (${typeof applicationOrDevice})`,
+			`Fetching application by id or name ${applicationOrDevice} (${typeof applicationOrDevice})`,
```

Comment:
Upper case 'ID' would look better?

Reply:
I prefer it lowercase but I don't mind, updated.

Reply:
You wrote here it's updated, but it isn't... :-)

Reply:
Sorry, I've updated in the comment above. Updating here too.